moviepy/audio/fx/audio_normalize.py

```diff
@@ -3,8 +3,19 @@
 
 @audio_video_fx
 def audio_normalize(clip):
-    """ Return an audio (or video) clip whose volume is normalized
-        to 0db."""
+    """ Return a clip whose volume is normalized to 0db.
+
+    Return an audio (or video) clip whose audio volume is normalized
+    so that the maximum volume is at 0db, the maximum achivable volume.
+
+    Examples
+    ========
+
+    >>> from moviepy.editor import *
+    >>> videoclip = VideoFileClip('myvideo.mp4').fx(afx.audio_normalize)
```

Comment:
Because of some 'moviepy.editor' `exec()` hackery, I believe that this can be replaced with
```
    >>> videoclip = VideoFileClip('myvideo.mp4').afx.audio_normalize()
```

Reply:
It doesn't seem to work that way.  I'm getting the error: `AttributeError: 'function' object has no attribute 'audio_normalize'`. I propose to leave it as is.

Reply:
Ok, you're correct on this one. I think that the api could be improved to work more like how I thought it did, but that would be in a new PR, perhaps in v2.0.